client/src/app/modules/project-page/components/project-page.component.spec.ts

```diff
@@ -60,9 +85,9 @@ describe('ProjectPageComponent', () => {
   xit('should show section to add new quickstarters when edit-button is clicked', () => {
     /* given */
     /* when */
-    spectator.click('[data-test-edit-btn]');
-    spectator.detectComponentChanges();
+    // spectator.click('[data-test-edit-btn]');
```

Comment:
these lines are commented our, they can be removed

Reply:
even better: the test is commented out (`xit()`) 😉